forms-flow-web/src/components/Form/Item/View.js

```diff
@@ -0,0 +1,209 @@
+import React, { useEffect } from "react";
+import { connect, useDispatch, useSelector } from "react-redux";
+import {
+  selectRoot,
+  resetSubmissions,
+  saveSubmission,
+  Form,
+  selectError,
+  Errors,
+  getForm,
+} from "react-formio";
+import { push } from "connected-react-router";
+import { Link } from "react-router-dom";
+
+import Loading from "../../../containers/Loading";
+import { getProcessReq } from "../../../apiManager/services/bpmServices";
+import {
+  setFormSubmissionError,
+  setFormSubmissionLoading,
+  setMaintainBPMFormPagination,
+} from "../../../actions/formActions";
+import {fetchEmployeeData} from "../../../apiManager/services/employeeDataService";
+import SubmissionError from "../../../containers/SubmissionError";
+import { applicationCreate } from "../../../apiManager/services/applicationServices";
+import LoadingOverlay from "react-loading-overlay";
+import { CUSTOM_EVENT_TYPE } from "../../ServiceFlow/constants/customEventTypes";
+import { toast } from "react-toastify";
+
+const View = React.memo((props) => {
+  const isFormSubmissionLoading = useSelector(
+    (state) => state.formDelete.isFormSubmissionLoading
+  );
+  const {
+    isAuthenticated,
+    hideComponents,
+    onSubmit,
+    onCustomEvent,
+    errors,
+    options,
+    form: { form, isActive, url },
+    getForm,
+    getEmployeeData,
+    employeeData,
+  } = props;
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!isAuthenticated) {
+      getForm();
+    }
+    getEmployeeData();
+    dispatch(setMaintainBPMFormPagination(true));
+
+  }, [getForm, getEmployeeData, isAuthenticated, dispatch]);
+
+  if (isActive) {
+    return (
+      <div data-testid='loading-view-component'>
+        <Loading />
+      </div>
+    );
+  }
+
+  return (
+    <div className='container'>
+      <div className='main-header'>
+        <SubmissionError
+          modalOpen={props.submissionError.modalOpen}
+          message={props.submissionError.message}
+          onConfirm={props.onConfirm}
+        ></SubmissionError>
+        {isAuthenticated ? (
+          <Link to='/form'>
+            <i className='fa fa-chevron-left fa-lg' />
+          </Link>
+        ) : null}
+        {/*          <span className="ml-3">
+            <img src="/form.svg" width="30" height="30" alt="form" />
+          </span>*/}
+        <h3 className='ml-3'>
+          <span className='task-head-details'>
+            <i className='fa fa-wpforms' aria-hidden='true' /> &nbsp; Forms /
+          </span>{" "}
+          {form.title}
+        </h3>
+      </div>
+      <Errors errors={errors} />
+      <LoadingOverlay
+        active={isFormSubmissionLoading}
+        spinner
+        text='Loading...'
+        className='col-12'
+      >
+        <div className='ml-4 mr-4'>
+          <Form
+            form={form}
+            submission={{ data: { ...employeeData.data} }}
```

Comment:
This is where `employeeData` actually passed as submission to the form to fill the fields that match the names.